loopbackStarter/manifest.yml

```diff
@@ -0,0 +1,10 @@
+applications:
+- path: .
+  memory: 256M
+  instances: 1
+  domain: mybluemix.net
+  name: loopback-starter-app
+  host: loopback-starter-app
+  disk_quota: 256M
```

Comment:
add under applications:
```
  services:
  - loopback-mongodb-starter
```

so if user forgot to create the service, the push will fail:
```
Could not find service loopback-mongodb-starter to bind to sai-app-lb
```